src/pages/instances/CreateInstance.tsx

```diff
@@ -259,14 +263,20 @@ const CreateInstance: FC = () => {
           operation.metadata.id,
           () => creationCompletedHandler(instanceName, shouldStart, isIsoImage),
           (msg) =>
-            notifyCreationFailed(new Error(msg), formUrl, values, "toast"),
+            notifyCreationFailed(
```

Comment:
I made this a bit more generic, if any error occurs the error notification would redirect to the relevant instance creation section. I think this is better than just redirecting back to the relevant section specifically for YAML errors